.gitignore

```diff
@@ -18,4 +18,6 @@ _build/
 dependency-reduced-pom.xml
 *.iml
 
+k8s_starter.py
+integration-tests/replica_test.py
```

Comment:
Remove this.

Reply:
Oops. Must have forgotten to save. Sorry about that.